admin/css/login.css

```diff
@@ -0,0 +1,121 @@
+.text-primary {
+  color: #204a87ff !important;
+}
+
+body,
+html {
+  font-family: "URW Gothic L" !important;
```

Comment:
Codacy found an issue: [Use of !important](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)